registration/models.py

```diff
@@ -265,12 +266,13 @@ def delete_expired_users(self):
         """
         for profile in self.all():
             try:
-                if profile.activation_key_expired():
+                if profile.activation_key_expired() and not profile.activated:
                     user = profile.user
-                    if not user.is_active:
-                        profile.delete()
-                        user.delete()
+                    logger.warning('Deleting expired Registration profile {} and user {}.'.format(profile, user))
```

Comment:
I don't believe `logging` is used anywhere else in this project; how will this impact users of this package? What happens if they don't have logging setup?

Reply:
It's not used elsewhere and I haven't noticed in a few other third party django apps I checked. But is there an alternative? We can suggest a default logging configuration in the docs and mention that it's necessary for the cron job unless there is a different approach we can take.

Reply:
I poked around a bit and convinced myself this should be good. :smile:

Other packages follow the same pattern.

Reply:
cool i'll merge in then!